<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <title>Lei Xiong</title>
  <meta name="description" content="">

  <!-- Fonts and Icons -->
  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Helvetica:300,400,500,700|Helvetica:100,300,400,500,700|Material+Icons" />

  <!-- CSS Files -->
  <link rel="stylesheet" href="/assets/css/all.min.css">
  <link rel="stylesheet" href="/assets/css/academicons.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/">
</head>
<body>
  <!-- Header -->
  <nav id="navbar" class="navbar fixed-top navbar-expand-md grey lighten-5 z-depth-1 navbar-light">
    <div class="container-fluid p-0">
      
        <div class="row ml-1 ml-sm-0">
          <span class="contact-icon text-center" style="line-height: 1em;">
            <a href="mailto:jsxlei@gmail.com"><i class="fa fa-envelope-square gm-icon"></i></a>
            <a href="https://scholar.google.com/citations?userid=ky6bUTgAAAAJ&hl&user=ky6bUTgAAAAJ&hl" target="_blank" title="Google Scholar"><i class="ai ai-google-scholar-square gs-icon"></i></a>
            <a href="https://www.github.com/jsxlei" target="_blank" title="GitHub"><i class="fab fa-github-square gh-icon"></i></a>
            <a href="https://www.linkedin.com/in/lei-xiong-6776a410b" target="_blank" title="LinkedIn"><i class="fab fa-linkedin li-icon"></i></a>
            <a href="https://www.facebook.com/jsxlei" target="_blank" title="Facebook"><i class="fab fa-facebook-square fb-icon"></i></a>
            <a href="https://twitter.com/leixiong_" target="_blank" title="Twitter"><i class="fab fa-twitter-square tw-icon"></i></a>
          </span>
        </div>
      
      <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <li class="nav-item navbar-active font-weight-bold">
            <a class="nav-link" href="/">
              Home
              
                <span class="sr-only">(current)</span>
              
            </a>
          </li>
          
            
          
            
          
            
          
            
          
            
          
        </ul>
      </div>
    </div>
  </nav>

  <!-- Scrolling Progress Bar -->
  <progress id="progress" value="0">
    <div class="progress-container">
      <span class="progress-bar"></span>
    </div>
  </progress>

  <!-- Content -->
  <div class="content">
    

<!-- Introduction -->

<div style="display: flex; flex-wrap: wrap;">
    <div class="text-justify p-0">
        <div id="sidebar" class="col-xs-12 col-sm-4 p-0 pt-2 pb-sm-2 pb-4 pl-sm-4 text-center" style="float: right;">
          <img class="profile-img img-responsive" src="/assets/img/prof_pic2.jpg" />
          <nav class="sidebar">
            <!-- <ul> -->
              <li class="font-weight-bold"><a href="/assets/pdf/CV_leixiong.pdf">Bio</a></li>
              <li class="font-weight-bold"><a href="#education">Education</a></li>
              <li class="font-weight-bold"><a href="#news">News</a></li>
              <li class="font-weight-bold"><a href="#publications">Publications</a></li>
              <li class="font-weight-bold"><a href="#softwares">Softwares</a></li>
              <li class="font-weight-bold"><a href="#honors and awards">Honors and Awards</a></li>
            <!-- </ul> -->
          </nav>
        </div>
        <h1 class="pb-3 title text-left font-weight-bold">Lei Xiong</h1>
          <p>
            <b>Postdoctoral Scholar</b> <br />
            <b>Genetics </b> <br /> 
            <b>Stanford University </b> <br />
          </p>
        <p> 
My research interests are centered on the development and application of innovative deep learning approaches to solve complex biological questions, with a primary focus on advancing our understanding of gene regulation and cellular diversity. To achieve this goal, I build models that can effectively capture and interpret complex features from biological datasets, which provides new insights into fundamental biological processes. Specifically, I will <b>Decipher Mechanisms of Macrophage Dynamics and Cellular Crosstalk in Cancer</b>. Through my work, I aim to drive significant advances in the field of single-cell multiomics and regulatory DNA sequence, which has the potential to contribute to a better understanding of human health and disease.
        </p>
        <p>
          Feel free to reach out by email <b>jsxlei at gmail.com</b> or other social media listed above!
        </p>
    </div>
</div>

<!-- Education-->
<div id="education" class="news mt-3 p-0">
  <h2 class="font-weight-bold">Education</h2>
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2015-2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Tsinghua University</b> <br />
            <b>Ph.D. in Computational Biology</b> <br />
            <b>Advisor</b>: Prof.<a href="http://life.tsinghua.edu.cn/lifeen/info/1034/1075.htm" target="_blank"><b>Qiangfeng Cliff Zhang</b></a> <br />
            <b>Thesis</b>: Artificial intelligence method for single-cell ATAC-seq data via feature extraction
        </p>
      </div>
    </div>
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2011-2015
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>University of Science and Technology of China (USTC)</b> <br />
            <b>B.S. in Biology, Shitsan Pai Talent Program in Life Sciences </b><br />
            <b>Advisor</b>: Prof.<a href="https://molbio.princeton.edu/people/nieng-yan" target="_blank"><b>Nieng Yan</b></a> <br />
            <b>Thesis</b>: Structure basis and transport mechanism of membrane protein GLUT3
        </p>
      </div>
    </div>
</div>

<!-- News -->
<div id="news" class="news mt-3 p-0">
  <h2 class="font-weight-bold">News</h2>
  
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Oct 23, 2024
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Our work <b><a href="https://www.nature.com/articles/s42256-024-00915-6">Deep learning prediction of ribosome profiling with Translatomer reveals translational regulation and interprets disease variants</a></b> was published on <i>Nature Machine Intelligence</i>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Sep 26, 2024
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Our work <b><a href="">A versatile informative diffusion model for single-cell ATAC-seq data generation and analysis</a></b> was accepted by NeurIPS 2024
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Jun 18, 2024
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Awarded <b>School of Medicine Deanâ€™s Postdoctoral Fellowship, Stanford</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Mar 25, 2024
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Join Prof.<a href="https://kundajelab.stanford.edu/" target="_blank"><b>Anshul Kundaje</b></a> lab in Genetics at Stanford, also co-supervised by Prof.<a href="https://bassiklab.net/" target="_blank"><b>Mike Bassik</b>.</a>
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Dec 16, 2023
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Our workshop paper <b><a href="https://openreview.net/pdf?id=KMtM5ZHxct">scCLIP: Multi-modal Single-cell Contrastive Learning Integration Pre-training</a></b> was presented on NeurIPS 2023 AI4Science workshop
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Oct 17, 2022
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Our work  <b><a href="https://www.nature.com/articles/s41467-022-33758-z" target="https://www.nature.com/articles/s41467-022-33758-z">Online single-cell data integration through projecting heterogeneous datasets into a common cell-embedding space</a></b> was published on <i>Nature Communications</i>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Dec 1, 2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Join Prof.<a href="http://compbio.mit.edu/" target="_blank"><b>Manolis Kellis</b></a> lab in CSAIL at MIT and Broad Institute of MIT and Harvard.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Jun 30, 2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Awarded <b>Outstanding Doctoral Dissertation Award of Tsinghua University</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Jan 13, 2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Awarded <b>Outstanding Graduate of Beijing</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Sep 14, 2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>PhD. thesis Defense.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Feb 1, 2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Our work SCALE was awarded <b>Top 10 Algorithms and Tools for Bioinformatics in China</b> in 2019.

</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Feb 1, 2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Our work SCALE was awarded <b>Top 10 Advances in Bioinformatics in China</b> in 2019
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Oct 8, 2019
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Our work  <b><a href="https://www.nature.com/articles/s41467-019-12630-7#citeas" target="https://www.nature.com/articles/s41467-019-12630-7#citeas">SCALE method for single-cell ATAC-seq analysis via latent feature extraction</a></b> was published on <i>Nature Communications</i>.
</p>
      </div>
    </div>
  
</div>

<!-- publications -->

<div id="publications" class="news mt-3 p-0">
  <h2 class="font-weight-bold">Publications</h2>
  <nobr><em><sup>*</sup></em></nobr> for equal authorship, <nobr><em><sup>#</sup></em></nobr> for corresponding authorship.
  
  <div class="row m-0 p-0" style="border-top: 1px solid #ddd; flex-direction: row-reverse;">
    <div class="col-sm-1 mt-2 p-0 pr-1">
      <h3 class="bibliography-year">2024</h3>
    </div>
    <div class="col-sm-11 p-0">
      <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/Translatomer.png" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <!-- <div id="He2024-dn" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.nature.com/articles/s42256-024-00915-6" target="_blank">Deep learning prediction of ribosome profiling with Translatomer
               reveals translational regulation and interprets disease variants.</a></h5>
      <div class="author">
        
          
            
              
                
                  <nobr>Jialin He<sup>*</sup>,</nobr>
                
              
            
          
        
          
            
              
                <nobr><em>Lei Xiong<sup>*,#</sup></em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Shaohui Shi,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Chengyu Li,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Kexuan Chen,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Qianchen Fang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Jiuhong Nan,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Ke Ding,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yuanhui Mao,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Carles A Boix,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Xinyang Hu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Manolis Kellis,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Jingyun Li,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Xushen Xiong<sup>#</sup></nobr>
                
              
            
          
        
      </div> -->
      <div id="He2024-dn" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.nature.com/articles/s42256-024-00915-6" target="_blank">Deep learning prediction of ribosome profiling with Translatomer
               reveals translational regulation and interprets disease variants.</a></h5>
        <div class="author">
          
            
              
                
                  
                    <nobr>Jialin He<sup>*</sup>,</nobr>
                  
                
              
            
          
            
              
                
                  <nobr><em>Lei Xiong<sup>*,#</sup></em>,</nobr>
                
              
            
          
            
              
                
                  
                    <nobr>Shaohui Shi,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Chengyu Li,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Kexuan Chen,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Qianchen Fang,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Jiuhong Nan,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Ke Ding,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Yuanhui Mao,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Carles A Boix,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Xinyang Hu,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Manolis Kellis,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Jingyun Li,</nobr>
                  
                
              
            
          
            
              
                and
                
                  
                    <nobr>Xushen Xiong<sup>#</sup></nobr>
                  
                
              
            
          
        </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>Nat. Mach. Intell.</i></em></nobr>
          
<!--            -->
          
            2024.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#He2024-dn-abstract" role="button" aria-expanded="false" aria-controls="He2024-dn-abstract">Abstract</a>
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="/assets/pdf/" target="_blank">PDF</a>
        
        
        
        
        
        
          <a class="fab fa-github gh-icon font-weight-light mr-1" href="https://github.com/jsxlei/Translatomer" target="_blank"></a>
        
        
        
        
        <!--  -->
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="He2024-dn-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            Gene expression involves transcription and translation. Despite large datasets
             and increasingly powerful methods devoted to calculating genetic variantsâ€™ effects 
             on transcription, discrepancy between messenger RNA and protein levels hinders the 
             systematic interpretation of the regulatory effects of disease-associated variants. 
             Accurate models of the sequence determinants of translation are needed to close this 
             gap and to interpret disease-associated variants that act on translation. Here we 
             present Translatomer, a multimodal transformer framework that predicts cell-type-specific 
             translation from messenger RNA expression and gene sequence. We train the Translatomer on 
             33 tissues and cell lines, and show that the inclusion of sequence improves the prediction 
             of ribosome profiling signal, indicating that the Translatomer captures sequence-dependent 
             translational regulatory information. The Translatomer achieves accuracies of 0.72 to 0.80 
             for the de novo prediction of cell-type-specific ribosome profiling. We develop an in silico 
             mutagenesis tool to estimate mutational effects on translation and demonstrate that variants 
             associated with translation regulation are evolutionarily constrained, both in the human 
             population and across species. In particular, we identify cell-type-specific translational 
             regulatory mechanisms independent of the expression quantitative trait loci for 3,041 
             non-coding and synonymous variants associated with complex diseases, including Alzheimerâ€™s 
             disease, schizophrenia and congenital heart disease. The Translatomer accurately models the 
             genetic underpinnings of translation, bridging the gap between messenger RNA and protein 
             levels as well as providing valuable mechanistic insights for uninterpreted disease variants.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li>
<li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/atac-diff.png" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <!-- <div id="Huang2024-py" class="col p-0">
        <h5 class="title mb-0"><a href="" target="_blank">A versatile informative diffusion model for single-cell
                   ATAC-seq data generation and analysis.</a></h5>
      <div class="author">
        
          
            
              
                
                  <nobr>Lei Huang<sup>*</sup>,</nobr>
                
              
            
          
        
          
            
              
                <nobr><em>Lei Xiong<sup>*</sup></em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Na Sun,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Zunpeng Liu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Ka-Chun Wong,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Manolis Kellis</nobr>
                
              
            
          
        
      </div> -->
      <div id="Huang2024-py" class="col p-0">
        <h5 class="title mb-0"><a href="" target="_blank">A versatile informative diffusion model for single-cell
                   ATAC-seq data generation and analysis.</a></h5>
        <div class="author">
          
            
              
                
                  
                    <nobr>Lei Huang<sup>*</sup>,</nobr>
                  
                
              
            
          
            
              
                
                  <nobr><em>Lei Xiong<sup>*</sup></em>,</nobr>
                
              
            
          
            
              
                
                  
                    <nobr>Na Sun,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Zunpeng Liu,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Ka-Chun Wong,</nobr>
                  
                
              
            
          
            
              
                and
                
                  
                    <nobr>Manolis Kellis</nobr>
                  
                
              
            
          
        </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>NeurIPS</i></em></nobr>
          
<!--            -->
          
            2024.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#Huang2024-py-abstract" role="button" aria-expanded="false" aria-controls="Huang2024-py-abstract">Abstract</a>
        
        
        
        
        
        
        
        
          <a class="fab fa-github gh-icon font-weight-light mr-1" href="" target="_blank"></a>
        
        
        
        
        <!--  -->
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="Huang2024-py-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            The rapid advancement of single-cell ATAC sequencing
                   (scATAC-seq) technologies holds great promise for
                   investigating the heterogeneity of epigenetic landscapes at
                   the cellular level. The amplification process in scATAC-seq
                   experiments often introduces noise due to dropout events,
                   which results in extreme sparsity that hinders accurate
                   analysis. Consequently, there is a significant demand for the
                   generation of high-quality scATAC-seq data in silico.
                   Furthermore, current methodologies are typically
                   task-specific, lacking a versatile framework capable of
                   handling multiple tasks within a single model. In this work,
                   we propose ATAC-Diff, a versatile framework, which is based
                   on a latent diffusion model conditioned on the latent
                   auxiliary variables to adapt for various tasks. ATAC-Diff is
                   the first diffusion model for the scATAC-seq data generation
                   and analysis, composed of auxiliary modules encoding the
                   latent high-level variables to enable the model to learn the
                   semantic information to sample high-quality data. Gaussian
                   Mixture Model (GMM) as the latent prior and auxiliary
                   decoder, the yield variables reserve the refined genomic
                   information beneficial for downstream analyses. Another
                   innovation is the incorporation of mutual information between
                   observed and hidden variables as a regularization term to
                   prevent the model from decoupling from latent variables.
                   Through extensive experiments, we demonstrate that ATAC-Diff
                   achieves high performance in both generation and analysis
                   tasks, outperforming state-of-the-art models.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li>
<li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/celegans_rna.png" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <!-- <div id="Chen2024-ls" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.tandfonline.com/doi/full/10.1080/15476286.2024.2332856" target="_blank">Tissue-specific silencing of integrated transgenes achieved
              through endogenous RNA interference in Caenorhabditis elegans.</a></h5>
      <div class="author">
        
          
            
              
                
                  <nobr>Siyu Chen,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Weihong Liu,</nobr>
                
              
            
          
        
          
            
              
                <nobr><em>Lei Xiong</em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Zhiju Tao,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Di Zhao</nobr>
                
              
            
          
        
      </div> -->
      <div id="Chen2024-ls" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.tandfonline.com/doi/full/10.1080/15476286.2024.2332856" target="_blank">Tissue-specific silencing of integrated transgenes achieved
              through endogenous RNA interference in Caenorhabditis elegans.</a></h5>
        <div class="author">
          
            
              
                
                  
                    <nobr>Siyu Chen,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Weihong Liu,</nobr>
                  
                
              
            
          
            
              
                
                  <nobr><em>Lei Xiong</em>,</nobr>
                
              
            
          
            
              
                
                  
                    <nobr>Zhiju Tao,</nobr>
                  
                
              
            
          
            
              
                and
                
                  
                    <nobr>Di Zhao</nobr>
                  
                
              
            
          
        </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>RNA Biol.</i></em></nobr>
          
<!--            -->
          
            2024.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#Chen2024-ls-abstract" role="button" aria-expanded="false" aria-controls="Chen2024-ls-abstract">Abstract</a>
        
        
        
        
        
        
        
        
        
        
        
        <!--  -->
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="Chen2024-ls-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            Transgene silencing is a common phenomenon observed in
              Caenorhabditis elegans, particularly in the germline, but the
              precise mechanisms underlying this process remain elusive. Through
              an analysis of the transcription factors profile of C. elegans, we
              discovered that the expression of several transgenic reporter
              lines exhibited tissue-specific silencing, specifically in the
              intestine of C. elegans. Notably, this silencing could be reversed
              in mutants defective in endogenous RNA interference (RNAi).
              Further investigation using knock-in strains revealed that these
              intestine-silent genes were indeed expressed in vivo, indicating
              that the organism itself regulates the intestine-specific
              silencing. This tissue-specific silencing appears to be mediated
              through the endo-RNAi pathway, with the main factors of this
              pathway, mut-2 and mut-16, are significantly enriched in the
              intestine. Additionally, histone modification factors, such as
              met-2, are involved in this silencing mechanism. Given the crucial
              role of the intestine in reproduction alongside the germline, the
              transgene silencing observed in the intestine reflects the
              self-protective mechanisms employed by the organisms. In summary,
              our study proposed that compared to other tissues, the transgenic
              silencing of intestine is specifically regulated by the endo-RNAi
              pathway.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li></ol>   
    </div>
  </div>
  
  <div class="row m-0 p-0" style="border-top: 1px solid #ddd; flex-direction: row-reverse;">
    <div class="col-sm-1 mt-2 p-0 pr-1">
      <h3 class="bibliography-year">2023</h3>
    </div>
    <div class="col-sm-11 p-0">
      <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/scCLIP.jpg" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <!-- <div id="Xiong2023-gw" class="col p-0">
        <h5 class="title mb-0"><a href="https://openreview.net/forum?id=KMtM5ZHxct" target="_blank">scCLIP: Multi-modal Single-cell Contrastive Learning
              Integration Pre-training.</a></h5>
      <div class="author">
        
          
            
              
                <nobr><em>Lei Xiong</em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Tianlong Chen,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Manolis Kellis</nobr>
                
              
            
          
        
      </div> -->
      <div id="Xiong2023-gw" class="col p-0">
        <h5 class="title mb-0"><a href="https://openreview.net/forum?id=KMtM5ZHxct" target="_blank">scCLIP: Multi-modal Single-cell Contrastive Learning
              Integration Pre-training.</a></h5>
        <div class="author">
          
            
              
                
                  <nobr><em>Lei Xiong</em>,</nobr>
                
              
            
          
            
              
                
                  
                    <nobr>Tianlong Chen,</nobr>
                  
                
              
            
          
            
              
                and
                
                  
                    <nobr>Manolis Kellis</nobr>
                  
                
              
            
          
        </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>NeurIPS AI for Science workshop</i></em></nobr>
          
<!--            -->
          
            2023.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#Xiong2023-gw-abstract" role="button" aria-expanded="false" aria-controls="Xiong2023-gw-abstract">Abstract</a>
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="/assets/pdf/scclip.pdf" target="_blank">PDF</a>
        
        
        
        
        
        
          <a class="fab fa-github gh-icon font-weight-light mr-1" href="https://github.com/jsxlei/scCLIP" target="_blank"></a>
        
        
        
        
        <!--  -->
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="Xiong2023-gw-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            Recent advances in multi-modal single-cell sequencing
              technologies enable the simultaneous profiling of chromatin
              accessibility and transcriptome in individual cells. Integration
              analysis of multi-modal single-cell data offers a more
              comprehensive understanding of the regulatory mechanisms linking
              chromatin status and gene expression, driving cellular processes
              and diseases. In order to acquire features that align peaks and
              genes within the same embedding space and facilitate seamless
              zero-shot transfer to new data, we introduced scCLIP (single-cell
              Contrastive Learning Integration Pretraining), a generalized
              multi-modal transformer model with contrastive learning. We show
              that this model outperforms other competing methods, and beyond
              this, scCLIP learns transferable features across modalities and
              generalizes to unseen datasets, which pose the great potential to
              bridge the vast number of unpaired unimodal datasets both
              existing and new data generated in the future. Specifically, we
              propose the first large-scale transformer model designed for
              single-cell ATAC-seq data by patching peaks across the genomes
              and representing each patch as a token. This innovative approach
              enables us effectively to address the scalability challenges
              posed by scATAC-seq, even when dealing with datasets of up to one
              million dimensions. Codes are provided at:
              https://github.com/jsxlei/scCLIP.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li></ol>   
    </div>
  </div>
  
  <div class="row m-0 p-0" style="border-top: 1px solid #ddd; flex-direction: row-reverse;">
    <div class="col-sm-1 mt-2 p-0 pr-1">
      <h3 class="bibliography-year">2022</h3>
    </div>
    <div class="col-sm-11 p-0">
      <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/scalex.jpg" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <!-- <div id="Xiong2022-cg" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.nature.com/articles/s41467-022-33758-z.pdf" target="_blank">Online single-cell data integration through projecting
              heterogeneous datasets into a common cell-embedding space.</a></h5>
      <div class="author">
        
          
            
              
                <nobr><em>Lei Xiong<sup>*</sup></em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Kang Tian<sup>*</sup>,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yuzhe Li,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Weixi Ning,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Xin Gao,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Qiangfeng Cliff Zhang<sup>#</sup></nobr>
                
              
            
          
        
      </div> -->
      <div id="Xiong2022-cg" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.nature.com/articles/s41467-022-33758-z.pdf" target="_blank">Online single-cell data integration through projecting
              heterogeneous datasets into a common cell-embedding space.</a></h5>
        <div class="author">
          
            
              
                
                  <nobr><em>Lei Xiong<sup>*</sup></em>,</nobr>
                
              
            
          
            
              
                
                  
                    <nobr>Kang Tian<sup>*</sup>,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Yuzhe Li,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Weixi Ning,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Xin Gao,</nobr>
                  
                
              
            
          
            
              
                and
                
                  
                    <nobr>Qiangfeng Cliff Zhang<sup>#</sup></nobr>
                  
                
              
            
          
        </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>Nat. Commun.</i></em></nobr>
          
<!--            -->
          
            2022.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#Xiong2022-cg-abstract" role="button" aria-expanded="false" aria-controls="Xiong2022-cg-abstract">Abstract</a>
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="/assets/pdf/scalex/scalex.pdf" target="_blank">PDF</a>
        
        
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="https://drive.google.com/file/d/1hO-De5mftCk62Kq4KVOY7vLHUFQ1QXiC/view?usp=share_link" target="_blank">Slides</a>
        
        
          <a class="fab fa-github gh-icon font-weight-light mr-1" href="https://github.com/jsxlei/SCALEX" target="_blank"></a>
        
        
        
        
        <!--  -->
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="Xiong2022-cg-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            Computational tools for integrative analyses of diverse
              single-cell experiments are facing formidable new challenges
              including dramatic increases in data scale, sample heterogeneity,
              and the need to informatively cross-reference new data with
              foundational datasets. Here, we present SCALEX, a deep-learning
              method that integrates single-cell data by projecting cells into
              a batch-invariant, common cell-embedding space in a truly online
              manner (i.e., without retraining the model). SCALEX substantially
              outperforms online iNMF and other state-of-the-art non-online
              integration methods on benchmark single-cell datasets of diverse
              modalities, (e.g., single-cell RNA sequencing, scRNA-seq,
              single-cell assay for transposase-accessible chromatin use
              sequencing, scATAC-seq), especially for datasets with partial
              overlaps, accurately aligning similar cell populations while
              retaining true biological differences. We showcase SCALEXâ€™s
              advantages by constructing continuously expandable single-cell
              atlases for human, mouse, and COVID-19 patients, each assembled
              from diverse data sources and growing with every new data. The
              online data integration capacity and superior performance makes
              SCALEX particularly appropriate for large-scale single-cell
              applications to build upon previous scientific insights.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li>
<li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/jem2022.png" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <!-- <div id="Zhang2022-ia" class="col p-0">
        <h5 class="title mb-0"><a href="https://doi.org/10.1084/jem.20211191" target="_blank">CD127 imprints functional heterogeneity to diversify monocyte
              responses in inflammatory diseases.</a></h5>
      <div class="author">
        
          
            
              
                
                  <nobr>Bin Zhang<sup>*</sup>,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yuan Zhang<sup>*</sup>,</nobr>
                
              
            
          
        
          
            
              
                <nobr><em>Lei Xiong</em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Yuzhe Li,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yunliang Zhang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Jiuliang Zhao,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Hui Jiang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Can Li,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yunqi Liu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Xindong Liu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Haofei Liu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yi-Fang Ping,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Qiangfeng Cliff Zhang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Zheng Zhang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Xiu-Wu Bian,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yan Zhao,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Xiaoyu Hu<sup>#</sup></nobr>
                
              
            
          
        
      </div> -->
      <div id="Zhang2022-ia" class="col p-0">
        <h5 class="title mb-0"><a href="https://doi.org/10.1084/jem.20211191" target="_blank">CD127 imprints functional heterogeneity to diversify monocyte
              responses in inflammatory diseases.</a></h5>
        <div class="author">
          
            
              
                
                  
                    <nobr>Bin Zhang<sup>*</sup>,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Yuan Zhang<sup>*</sup>,</nobr>
                  
                
              
            
          
            
              
                
                  <nobr><em>Lei Xiong</em>,</nobr>
                
              
            
          
            
              
                
                  
                    <nobr>Yuzhe Li,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Yunliang Zhang,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Jiuliang Zhao,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Hui Jiang,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Can Li,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Yunqi Liu,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Xindong Liu,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Haofei Liu,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Yi-Fang Ping,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Qiangfeng Cliff Zhang,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Zheng Zhang,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Xiu-Wu Bian,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Yan Zhao,</nobr>
                  
                
              
            
          
            
              
                and
                
                  
                    <nobr>Xiaoyu Hu<sup>#</sup></nobr>
                  
                
              
            
          
        </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>J. Exp. Med.</i></em></nobr>
          
<!--            -->
          
            2022.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#Zhang2022-ia-abstract" role="button" aria-expanded="false" aria-controls="Zhang2022-ia-abstract">Abstract</a>
        
        
        
        
        
        
        
        
        
        
        
        <!--  -->
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="Zhang2022-ia-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            Inflammatory monocytes are key mediators of acute and chronic
              inflammation; yet, their functional diversity remains obscure.
              Single-cell transcriptome analyses of human inflammatory
              monocytes from COVID-19 and rheumatoid arthritis patients
              revealed a subset of cells positive for CD127, an IL-7 receptor
              subunit, and such positivity rendered otherwise inert monocytes
              responsive to IL-7. Active IL-7 signaling engaged epigenetically
              coupled, STAT5-coordinated transcriptional programs to restrain
              inflammatory gene expression, resulting in inverse correlation
              between CD127 expression and inflammatory phenotypes in a
              seemingly homogeneous monocyte population. In COVID-19 and
              rheumatoid arthritis, CD127 marked a subset of
              monocytes/macrophages that retained hypoinflammatory phenotypes
              within the highly inflammatory tissue environments. Furthermore,
              generation of an integrated expression atlas revealed unified
              features of human inflammatory monocytes across different
              diseases and different tissues, exemplified by those of the
              CD127high subset. Overall, we phenotypically and molecularly
              characterized CD127-imprinted functional heterogeneity of human
              inflammatory monocytes with direct relevance for inflammatory
              diseases.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li></ol>   
    </div>
  </div>
  
  <div class="row m-0 p-0" style="border-top: 1px solid #ddd; flex-direction: row-reverse;">
    <div class="col-sm-1 mt-2 p-0 pr-1">
      <h3 class="bibliography-year">2019</h3>
    </div>
    <div class="col-sm-11 p-0">
      <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/scale.jpg" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <!-- <div id="Xiong2019-xm" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.nature.com/articles/nature14655.pdf" target="_blank">SCALE method for single-cell ATAC-seq analysis via latent
              feature extraction.</a></h5>
      <div class="author">
        
          
            
              
                <nobr><em>Lei Xiong</em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Kui Xu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Kang Tian,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yanqiu Shao,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Lei Tang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Ge Gao,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Michael Zhang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Tao Jiang,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Qiangfeng Cliff Zhang<sup>#</sup></nobr>
                
              
            
          
        
      </div> -->
      <div id="Xiong2019-xm" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.nature.com/articles/nature14655.pdf" target="_blank">SCALE method for single-cell ATAC-seq analysis via latent
              feature extraction.</a></h5>
        <div class="author">
          
            
              
                
                  <nobr><em>Lei Xiong</em>,</nobr>
                
              
            
          
            
              
                
                  
                    <nobr>Kui Xu,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Kang Tian,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Yanqiu Shao,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Lei Tang,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Ge Gao,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Michael Zhang,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Tao Jiang,</nobr>
                  
                
              
            
          
            
              
                and
                
                  
                    <nobr>Qiangfeng Cliff Zhang<sup>#</sup></nobr>
                  
                
              
            
          
        </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>Nat. Commun.</i></em></nobr>
          
<!--            -->
          
            2019.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#Xiong2019-xm-abstract" role="button" aria-expanded="false" aria-controls="Xiong2019-xm-abstract">Abstract</a>
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="/assets/pdf/scale/scale.pdf" target="_blank">PDF</a>
        
        
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="https://drive.google.com/file/d/1BdIxheIhiqsVn9w0PHkdzTesSsCbYb8G/view?usp=share_link" target="_blank">Slides</a>
        
        
          <a class="fab fa-github gh-icon font-weight-light mr-1" href="https://github.com/jsxlei/SCALE" target="_blank"></a>
        
        
        
        
        <!--  -->
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="Xiong2019-xm-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            Single-cell ATAC-seq (scATAC-seq) profiles the chromatin
              accessibility landscape at single cell level, thus revealing
              cell-to-cell variability in gene regulation. However, the high
              dimensionality and sparsity of scATAC-seq data often complicate
              the analysis. Here, we introduce a method for analyzing
              scATAC-seq data, called Single-Cell ATAC-seq analysis via Latent
              feature Extraction (SCALE). SCALE combines a deep generative
              framework and a probabilistic Gaussian Mixture Model to learn
              latent features that accurately characterize scATAC-seq data. We
              validate SCALE on datasets generated on different platforms with
              different protocols, and having different overall data qualities.
              SCALE substantially outperforms the other tools in all aspects of
              scATAC-seq data analysis, including visualization, clustering,
              and denoising and imputation. Importantly, SCALE also generates
              interpretable features that directly link to cell populations,
              and can potentially reveal batch effects in scATAC-seq
              experiments.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li></ol>   
    </div>
  </div>
  
  <div class="row m-0 p-0" style="border-top: 1px solid #ddd; flex-direction: row-reverse;">
    <div class="col-sm-1 mt-2 p-0 pr-1">
      <h3 class="bibliography-year">2015</h3>
    </div>
    <div class="col-sm-11 p-0">
      <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/glut3.png" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <!-- <div id="Deng2015-vs" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.nature.com/articles/nature14655.pdf" target="_blank">Molecular basis of ligand recognition and transport by glucose
              transporters.</a></h5>
      <div class="author">
        
          
            
              
                
                  <nobr>Dong Deng<sup>*</sup>,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Pengcheng Sun<sup>*</sup>,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Chuangye Yan,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Meng Ke,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Xin Jiang,</nobr>
                
              
            
          
        
          
            
              
                <nobr><em>Lei Xiong</em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Wenlin Ren,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Kunio Hirata,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Masaki Yamamoto,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Shilong Fan,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Nieng Yan<sup>#</sup></nobr>
                
              
            
          
        
      </div> -->
      <div id="Deng2015-vs" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.nature.com/articles/nature14655.pdf" target="_blank">Molecular basis of ligand recognition and transport by glucose
              transporters.</a></h5>
        <div class="author">
          
            
              
                
                  
                    <nobr>Dong Deng<sup>*</sup>,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Pengcheng Sun<sup>*</sup>,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Chuangye Yan,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Meng Ke,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Xin Jiang,</nobr>
                  
                
              
            
          
            
              
                
                  <nobr><em>Lei Xiong</em>,</nobr>
                
              
            
          
            
              
                
                  
                    <nobr>Wenlin Ren,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Kunio Hirata,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Masaki Yamamoto,</nobr>
                  
                
              
            
          
            
              
                
                  
                    <nobr>Shilong Fan,</nobr>
                  
                
              
            
          
            
              
                and
                
                  
                    <nobr>Nieng Yan<sup>#</sup></nobr>
                  
                
              
            
          
        </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>Nature</i></em></nobr>
          
<!--            -->
          
            2015.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#Deng2015-vs-abstract" role="button" aria-expanded="false" aria-controls="Deng2015-vs-abstract">Abstract</a>
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="/assets/pdf/glut3/glut3.pdf" target="_blank">PDF</a>
        
        
        
        
        
        
        
        
        
        <!--  -->
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="Deng2015-vs-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            The major facilitator superfamily glucose transporters,
              exemplified by human GLUT1-4, have been central to the study of
              solute transport. Using lipidic cubic phase crystallization and
              microfocus X-ray diffraction, we determined the structure of
              human GLUT3 in complex with D-glucose at 1.5 Ã… resolution in
              an outward-occluded conformation. The high-resolution structure
              allows discrimination of both Î±- and Î²-anomers of
              D-glucose. Two additional structures of GLUT3 bound to the
              exofacial inhibitor maltose were obtained at 2.6 Ã… in the
              outward-open and 2.4 Ã… in the outward-occluded states. In all
              three structures, the ligands are predominantly coordinated by
              polar residues from the carboxy terminal domain. Conformational
              transition from outward-open to outward-occluded entails a
              prominent local rearrangement of the extracellular part of
              transmembrane segment TM7. Comparison of the outward-facing GLUT3
              structures with the inward-open GLUT1 provides insights into the
              alternating access cycle for GLUTs, whereby the C-terminal domain
              provides the primary substrate-binding site and the
              amino-terminal domain undergoes rigid-body rotation with respect
              to the C-terminal domain. Our studies provide an important
              framework for the mechanistic and kinetic understanding of GLUTs
              and shed light on structure-guided ligand design.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li></ol>   
    </div>
  </div>
  
</div>

<!--  @*[selected=true]*   -->

<!-- Software -->
<div id="software" class="news mt-3 p-0">
  <h2 class="font-weight-bold">Softwares</h2>
  <div id="softwares" class="row mt-2 pt-3" style="overflow: visible !important;">
    
    
      <div class="project-card">
        
          <a href="/softwares/4_translatomer/">
        
          <div class="card">
            <img class="card-img-top" src="/assets/img/publications/translatomer.png" alt="project thumbnail" height="150" />
            <div class="card-body">
              <h5 class="card-title">Translatomer</h5>
              <p class="card-text">predicting ribosome profiling reveals translational regulation and interprets disease variants</p>
              <p>
                <!-- <a href="https://github.com/Translatomer" target="_blank"><i class="fab fa-github gh-icon"></i></a> -->
                <a href="https://github.com/jsxlei/Translatomer/stargazers"><img src="https://img.shields.io/github/stars/jsxlei/Translatomer?logo=GitHub&amp;color=yellow" alt="Stars" /></a>
                <a href="https://pypi.org/software/Translatomer"><img src="https://img.shields.io/pypi/v/Translatomer.svg" alt="PyPI" /></a>
                <a href="https://pepy.tech/software/Translatomer"><img src="https://pepy.tech/badge/Translatomer" alt="Downloads" /></a>
              </p>
              <div class="row ml-1 mr-1 p-0">
                
              </div>
            </div>
          </div>
        </a>
      </div>
    
      <div class="project-card">
        
          <a href="/softwares/3_scclip/">
        
          <div class="card">
            <img class="card-img-top" src="/assets/img/publications/scCLIP.jpg" alt="project thumbnail" height="150" />
            <div class="card-body">
              <h5 class="card-title">scCLIP</h5>
              <p class="card-text">Multi-modal Single-cell Integration</p>
              <p>
                <!-- <a href="https://github.com/scCLIP" target="_blank"><i class="fab fa-github gh-icon"></i></a> -->
                <a href="https://github.com/jsxlei/scCLIP/stargazers"><img src="https://img.shields.io/github/stars/jsxlei/scCLIP?logo=GitHub&amp;color=yellow" alt="Stars" /></a>
                <a href="https://pypi.org/software/scCLIP"><img src="https://img.shields.io/pypi/v/scCLIP.svg" alt="PyPI" /></a>
                <a href="https://pepy.tech/software/scCLIP"><img src="https://pepy.tech/badge/scCLIP" alt="Downloads" /></a>
              </p>
              <div class="row ml-1 mr-1 p-0">
                
              </div>
            </div>
          </div>
        </a>
      </div>
    
      <div class="project-card">
        
          <a href="/softwares/2_scalex/">
        
          <div class="card">
            <img class="card-img-top" src="/assets/img/publications/scalex.jpg" alt="project thumbnail" height="150" />
            <div class="card-body">
              <h5 class="card-title">SCALEX</h5>
              <p class="card-text">Online integration and projection of atlas-level single-cell datasets</p>
              <p>
                <!-- <a href="https://github.com/SCALEX" target="_blank"><i class="fab fa-github gh-icon"></i></a> -->
                <a href="https://github.com/jsxlei/SCALEX/stargazers"><img src="https://img.shields.io/github/stars/jsxlei/SCALEX?logo=GitHub&amp;color=yellow" alt="Stars" /></a>
                <a href="https://pypi.org/software/SCALEX"><img src="https://img.shields.io/pypi/v/SCALEX.svg" alt="PyPI" /></a>
                <a href="https://pepy.tech/software/SCALEX"><img src="https://pepy.tech/badge/SCALEX" alt="Downloads" /></a>
              </p>
              <div class="row ml-1 mr-1 p-0">
                
              </div>
            </div>
          </div>
        </a>
      </div>
    
      <div class="project-card">
        
          <a href="/softwares/1_scale/">
        
          <div class="card">
            <img class="card-img-top" src="/assets/img/publications/scale.jpg" alt="project thumbnail" height="150" />
            <div class="card-body">
              <h5 class="card-title">SCALE</h5>
              <p class="card-text">single-cell ATAC-seq clustering and imputation</p>
              <p>
                <!-- <a href="https://github.com/SCALE" target="_blank"><i class="fab fa-github gh-icon"></i></a> -->
                <a href="https://github.com/jsxlei/SCALE/stargazers"><img src="https://img.shields.io/github/stars/jsxlei/SCALE?logo=GitHub&amp;color=yellow" alt="Stars" /></a>
                <a href="https://pypi.org/software/SCALE"><img src="https://img.shields.io/pypi/v/SCALE.svg" alt="PyPI" /></a>
                <a href="https://pepy.tech/software/SCALE"><img src="https://pepy.tech/badge/SCALE" alt="Downloads" /></a>
              </p>
              <div class="row ml-1 mr-1 p-0">
                
              </div>
            </div>
          </div>
        </a>
      </div>
    
  </div>
<div class="news mt-3 p-0">


<!-- Honors and Awards-->
<div id="honors and awards" class="news mt-3 p-0">
  <h2 class="font-weight-bold">Honors and Awards</h2>
  
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2024
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>School of Medicine Deanâ€™s Postdoctoral Fellowship, Stanford</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Outstanding Doctoral Dissertation of Tsinghua University (Top 5%)</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Outstanding Graduate of Beijing (Top 5%)</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Top 10 Advances in Bioinformatics in China in 2019</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Top 10 Algorithms and Tools for Bioinformatics in China in 2019</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2019
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Outstanding Fellowship, Advanced Innovation Center of Structure Biology, Tsinghua</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2016
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Innovation Fellowship, Advanced Innovation Center of Structural Biology, Tsinghua</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2013
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>iGEM Gold Medal, USTC-China</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2013
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>2012-2013 Student Scholarship, USTC</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2012
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>2011-2012 Student Scholarship, USTC</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2011
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Freshman Scholarship, USTC</b>.
</p>
      </div>
    </div>
  
</div>
</div></div>


  </div>

  <!-- Footer -->
  <footer>
    &copy; Copyright 2024 Lei Xiong.
    
    
  </footer>

  <!-- Core JavaScript Files -->
  <script src="/assets/js/jquery.min.js" type="text/javascript"></script>
  <script src="/assets/js/popper.min.js" type="text/javascript"></script>
  <script src="/assets/js/bootstrap.min.js" type="text/javascript"></script>
  <script src="/assets/js/mdb.min.js" type="text/javascript"></script>
  <script async="" src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="sha384-GNFwBvfVxBkLMJpYMOABq3c+d3KnQxudP/mGPkzpZSTYykLBNsZEnG2D9G/X/+7D" crossorigin="anonymous"></script>
  <script src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML"></script>
  <script src="/assets/js/common.js"></script>

  <!-- Scrolling Progress Bar -->
  <script type="text/javascript">
    $(document).ready(function() {
      var navbarHeight = $('#navbar').outerHeight(true);
      $('body').css({ 'padding-top': navbarHeight });
      $('progress-container').css({ 'padding-top': navbarHeight });
      var progressBar = $('#progress');
      progressBar.css({ 'top': navbarHeight });
      var getMax = function() { return $(document).height() - $(window).height(); }
      var getValue = function() { return $(window).scrollTop(); }   
      // Check if the browser supports the progress element.
      if ('max' in document.createElement('progress')) {
        // Set the 'max' attribute for the first time.
        progressBar.attr({ max: getMax() });
        progressBar.attr({ value: getValue() });
    
        $(document).on('scroll', function() {
          // On scroll only the 'value' attribute needs to be calculated.
          progressBar.attr({ value: getValue() });
        });

        $(window).resize(function() {
          var navbarHeight = $('#navbar').outerHeight(true);
          $('body').css({ 'padding-top': navbarHeight });
          $('progress-container').css({ 'padding-top': navbarHeight });
          progressBar.css({ 'top': navbarHeight });
          // On resize, both the 'max' and 'value' attributes need to be calculated.
          progressBar.attr({ max: getMax(), value: getValue() });
        });
      } else {
        var max = getMax(), value, width;
        var getWidth = function() {
          // Calculate the window width as a percentage.
          value = getValue();
          width = (value/max) * 100;
          width = width + '%';
          return width;
        }
        var setWidth = function() { progressBar.css({ width: getWidth() }); };
        setWidth();
        $(document).on('scroll', setWidth);
        $(window).on('resize', function() {
          // Need to reset the 'max' attribute.
          max = getMax();
          setWidth();
        });
      }
    });
  </script>

  <!-- Code Syntax Highlighting -->
  <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet">
  <script src="/assets/js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>

  <!-- Script Used for Randomizing the Projects Order -->
  <!-- <script type="text/javascript">
    $.fn.shuffleChildren = function() {
      $.each(this.get(), function(index, el) {
        var $el = $(el);
        var $find = $el.children();

        $find.sort(function() {
          return 0.5 - Math.random();
        });

        $el.empty();
        $find.appendTo($el);
      });
    };
    $("#projects").shuffleChildren();
  </script> -->

  <!-- Project Cards Layout -->
  <script type="text/javascript">
    var $grid = $('#projects');

    // $grid.masonry({ percentPosition: true });
    // $grid.masonry('layout');

    // Trigger after images load.
    $grid.imagesLoaded().progress(function() {
      $grid.masonry({ percentPosition: true });
      $grid.masonry('layout');
    });
  </script>

  <!-- Enable Tooltips -->
  <script type="text/javascript">
    $(function () {
      $('[data-toggle="tooltip"]').tooltip()
    })
  </script>

  <!-- Google Analytics -->
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    ga('create', 'UA-197596889-1', 'auto');
    ga('send', 'pageview');
  </script>
    
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZXQXWK2E6C"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-ZXQXWK2E6C');
  </script>
    
  <!-- Baidu Tongji -->  
  <script>
    var _hmt = _hmt || [];
    (function() {
    var hm = document.createElement("script");
    hm.src = "https://hm.baidu.com/hm.js?fc5a69a8465cb0fd34201c2fab055fc4";
    var s = document.getElementsByTagName("script")[0]; 
    s.parentNode.insertBefore(hm, s);
    })();
  </script>

</body>
</html>
