<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <title>Lei Xiong | SCALEX</title>
  <meta name="description" content="">

  <!-- Fonts and Icons -->
  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Helvetica:300,400,500,700|Helvetica:100,300,400,500,700|Material+Icons" />

  <!-- CSS Files -->
  <link rel="stylesheet" href="/assets/css/all.min.css">
  <link rel="stylesheet" href="/assets/css/academicons.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/softwares/2_scalex/">
</head>
<body>
  <!-- Header -->
  <nav id="navbar" class="navbar fixed-top navbar-expand-md grey lighten-5 z-depth-1 navbar-light">
    <div class="container-fluid p-0">
      
        <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Lei</span> Xiong</a>
      
      <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <li class="nav-item ">
            <a class="nav-link" href="/">
              Home
              
            </a>
          </li>
          
            
          
            
          
            
          
            
          
            
          
        </ul>
      </div>
    </div>
  </nav>

  <!-- Scrolling Progress Bar -->
  <progress id="progress" value="0">
    <div class="progress-container">
      <span class="progress-bar"></span>
    </div>
  </progress>

  <!-- Content -->
  <div class="content">
    
  <h1>SCALEX</h1>
  <h6>Online integration and projection of atlas-level single-cell datasets</h6>


<p><a href="https://github.com/jsxlei/scalex/stargazers"><img src="https://img.shields.io/github/stars/jsxlei/scalex?logo=GitHub&amp;color=yellow" alt="Stars" /></a>
<a href="https://pypi.org/project/scalex"><img src="https://img.shields.io/pypi/v/scalex.svg" alt="PyPI" /></a>
<a href="https://scalex.readthedocs.io/en/latest/?badge=stable"><img src="https://readthedocs.org/projects/scalex/badge/?version=latest" alt="Documentation Status" /></a>
<a href="https://pepy.tech/project/scalex"><img src="https://pepy.tech/badge/scalex" alt="Downloads" /></a></p>
<h1 id="online-single-cell-data-integration-through-projecting-heterogeneous-datasets-into-a-common-cell-embedding-space"><a href="https://www.biorxiv.org/content/10.1101/2021.04.06.438536v2">Online single-cell data integration through projecting heterogeneous datasets into a common cell-embedding space</a></h1>

<p><img src="docs/source/_static/img/scalex.jpg" alt="" /></p>

<h2 id="documentation"><a href="https://scalex.readthedocs.io/en/latest/index.html">Documentation</a></h2>

<h2 id="installation">Installation</h2>
<h4 id="install-from-pypi">install from PyPI</h4>

<div class="highlighter-rouge"><pre class="highlight"><code>pip install scalex
</code></pre>
</div>

<h4 id="install-from-github">install from GitHub</h4>

<div class="highlighter-rouge"><pre class="highlight"><code>git clone git://github.com/jsxlei/scalex.git
cd scalex
python setup.py install
</code></pre>
</div>

<p>SCALEX is implemented in <a href="https://pytorch.org/">Pytorch</a> framework.<br />
SCALEX can be run on CPU devices, and running SCALEX on GPU devices if available is recommended.</p>

<h2 id="quick-start">Quick Start</h2>

<p>SCALEX can both used under command line and API function in jupyter notebook</p>

<h3 id="1-command-line">1. Command line</h3>
<h4 id="standard-usage">Standard usage</h4>

<div class="highlighter-rouge"><pre class="highlight"><code>SCALEX.py --data_list data1 data2 dataN --batch_categories batch_name1 batch_name2 batch_nameN 
</code></pre>
</div>

<p><code class="highlighter-rouge">--data_list</code>: data path of each batch of single-cell dataset, use <code class="highlighter-rouge">-d</code> for short</p>

<p><code class="highlighter-rouge">--batch_categories</code>: name of each batch, batch_categories will range from 0 to N if not specified</p>

<h4 id="use-h5ad-file-storing-anndata-as-input-one-or-multiple-files">Use h5ad file storing <code class="highlighter-rouge">anndata</code> as input, one or multiple files</h4>

<div class="highlighter-rouge"><pre class="highlight"><code>SCALEX.py --datalist &lt;filename.h5ad&gt;
</code></pre>
</div>

<h4 id="specify-batch-in-anadataobs-using---batch_name-if-only-one-concatenated-h5ad-file-provided-batch_name-can-be-eg-conditions-samples-assays-or-patients-default-is-batch">Specify batch in <code class="highlighter-rouge">anadata.obs</code> using <code class="highlighter-rouge">--batch_name</code> if only one concatenated h5ad file provided, batch_name can be e.g. conditions, samples, assays or patients, default is <code class="highlighter-rouge">batch</code></h4>

<div class="highlighter-rouge"><pre class="highlight"><code>SCALEX.py --datalist &lt;filename.h5ad&gt; --batch_name &lt;specific_batch_name&gt;
</code></pre>
</div>

<h4 id="integrate-heterogenous-scatac-seq-datasets-add-option---profile-atac">Integrate heterogenous scATAC-seq datasets, add option <code class="highlighter-rouge">--profile</code> ATAC</h4>

<div class="highlighter-rouge"><pre class="highlight"><code>SCALEX.py --datalist &lt;filename.h5ad&gt; --profile ATAC
</code></pre>
</div>

<h4 id="inputation-simultaneously-along-with-integration-add-option---impute-results-are-stored-at-anndatalayersimpute">Inputation simultaneously along with Integration, add option <code class="highlighter-rouge">--impute</code>, results are stored at anndata.layers[‘impute’]</h4>

<div class="highlighter-rouge"><pre class="highlight"><code>SCALEX.py --datalist &lt;atac_filename.h5ad&gt; --profile ATAC --impute
</code></pre>
</div>

<h4 id="custom-features-through---n_top_features-a-filename-contains-features-in-one-column-format-read">Custom features through <code class="highlighter-rouge">--n_top_features</code> a filename contains features in one column format read</h4>

<div class="highlighter-rouge"><pre class="highlight"><code>SCALEX.py --datalist &lt;filename.h5ad&gt; --n_top_features features.txt
</code></pre>
</div>

<h4 id="option">Option</h4>

<ul>
  <li>–<strong>data_list</strong><br />
      A list of matrices file (each as a <code class="highlighter-rouge">batch</code>) or a single batch/batch-merged file.</li>
  <li>–<strong>batch_categories</strong><br />
      Categories for the batch annotation. By default, use increasing numbers if not given</li>
  <li>–<strong>batch_name</strong><br />
      Use this annotation in anndata.obs as batches for training model. Default: ‘batch’.</li>
  <li>–<strong>profile</strong><br />
      Specify the single-cell profile, RNA or ATAC. Default: RNA.</li>
  <li>–<strong>min_features</strong><br />
      Filtered out cells that are detected in less than min_features. Default: 600 for RNA, 100 for ATAC.</li>
  <li>–<strong>min_cells</strong><br />
      Filtered out genes that are detected in less than min_cells. Default: 3.</li>
  <li>–<strong>n_top_features</strong><br />
      Number of highly-variable genes to keep. Default: 2000 for RNA, 30000 for ATAC.</li>
  <li>–<strong>outdir</strong><br />
      Output directory. Default: ‘output/’.</li>
  <li>–<strong>projection</strong><br />
      Use for new dataset projection. Input the folder containing the pre-trained model. Default: None.</li>
  <li>–<strong>impute</strong><br />
      If True, calculate the imputed gene expression and store it at adata.layers[‘impute’]. Default: False.</li>
  <li>–<strong>chunk_size</strong><br />
      Number of samples from the same batch to transform. Default: 20000.</li>
  <li>–<strong>ignore_umap</strong><br />
      If True, do not perform UMAP for visualization and leiden for clustering. Default: False.</li>
  <li>–<strong>join</strong><br />
      Use intersection (‘inner’) or union (‘outer’) of variables of different batches.</li>
  <li>–<strong>batch_key</strong><br />
      Add the batch annotation to obs using this key. By default, batch_key=’batch’.</li>
  <li>–<strong>batch_size</strong><br />
      Number of samples per batch to load. Default: 64.</li>
  <li>–<strong>lr</strong><br />
      Learning rate. Default: 2e-4.</li>
  <li>–<strong>max_iteration</strong><br />
      Max iterations for training. Training one batch_size samples is one iteration. Default: 30000.</li>
  <li>–<strong>seed</strong><br />
      Random seed for torch and numpy. Default: 124.</li>
  <li>–<strong>gpu</strong><br />
      Index of GPU to use if GPU is available. Default: 0.</li>
  <li>–<strong>verbose</strong><br />
      Verbosity, True or False. Default: False.</li>
</ul>

<h4 id="output">Output</h4>
<p>Output will be saved in the output folder including:</p>
<ul>
  <li><strong>checkpoint</strong>:  saved model to reproduce results cooperated with option –checkpoint or -c</li>
  <li><strong><a href="https://anndata.readthedocs.io/en/stable/anndata.AnnData.html#anndata.AnnData">adata.h5ad</a></strong>:  preprocessed data and results including, latent, clustering and imputation</li>
  <li><strong>umap.png</strong>:  UMAP visualization of latent representations of cells</li>
  <li><strong>log.txt</strong>:  log file of training process</li>
</ul>

<h4 id="useful-options">Useful options</h4>
<ul>
  <li>output folder for saveing results: [-o] or [–outdir]</li>
  <li>filter rare genes, default 3: [–min_cells]</li>
  <li>filter low quality cells, default 600: [–min_features]</li>
  <li>select the number of highly variable genes, keep all genes with -1, default 2000: [–n_top_featuress]</li>
</ul>

<h4 id="help">Help</h4>
<p>Look for more usage of SCALEX</p>

<div class="highlighter-rouge"><pre class="highlight"><code>SCALEX.py --help 
</code></pre>
</div>

<h3 id="2-api-function">2. API function</h3>

<div class="highlighter-rouge"><pre class="highlight"><code>from scalex import SCALEX
adata = SCALEX(data_list, batch_categories)
</code></pre>
</div>

<p>Function of parameters are similar to command line options.
Output is a Anndata object for further analysis with scanpy.</p>

<h2 id="tutorial"><a href="https://scalex.readthedocs.io/en/latest/tutorial/index.html">Tutorial</a></h2>


  </div>

  <!-- Footer -->
  <footer>
    &copy; Copyright 2024 Lei Xiong.
    
    
  </footer>

  <!-- Core JavaScript Files -->
  <script src="/assets/js/jquery.min.js" type="text/javascript"></script>
  <script src="/assets/js/popper.min.js" type="text/javascript"></script>
  <script src="/assets/js/bootstrap.min.js" type="text/javascript"></script>
  <script src="/assets/js/mdb.min.js" type="text/javascript"></script>
  <script async="" src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="sha384-GNFwBvfVxBkLMJpYMOABq3c+d3KnQxudP/mGPkzpZSTYykLBNsZEnG2D9G/X/+7D" crossorigin="anonymous"></script>
  <script src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML"></script>
  <script src="/assets/js/common.js"></script>

  <!-- Scrolling Progress Bar -->
  <script type="text/javascript">
    $(document).ready(function() {
      var navbarHeight = $('#navbar').outerHeight(true);
      $('body').css({ 'padding-top': navbarHeight });
      $('progress-container').css({ 'padding-top': navbarHeight });
      var progressBar = $('#progress');
      progressBar.css({ 'top': navbarHeight });
      var getMax = function() { return $(document).height() - $(window).height(); }
      var getValue = function() { return $(window).scrollTop(); }   
      // Check if the browser supports the progress element.
      if ('max' in document.createElement('progress')) {
        // Set the 'max' attribute for the first time.
        progressBar.attr({ max: getMax() });
        progressBar.attr({ value: getValue() });
    
        $(document).on('scroll', function() {
          // On scroll only the 'value' attribute needs to be calculated.
          progressBar.attr({ value: getValue() });
        });

        $(window).resize(function() {
          var navbarHeight = $('#navbar').outerHeight(true);
          $('body').css({ 'padding-top': navbarHeight });
          $('progress-container').css({ 'padding-top': navbarHeight });
          progressBar.css({ 'top': navbarHeight });
          // On resize, both the 'max' and 'value' attributes need to be calculated.
          progressBar.attr({ max: getMax(), value: getValue() });
        });
      } else {
        var max = getMax(), value, width;
        var getWidth = function() {
          // Calculate the window width as a percentage.
          value = getValue();
          width = (value/max) * 100;
          width = width + '%';
          return width;
        }
        var setWidth = function() { progressBar.css({ width: getWidth() }); };
        setWidth();
        $(document).on('scroll', setWidth);
        $(window).on('resize', function() {
          // Need to reset the 'max' attribute.
          max = getMax();
          setWidth();
        });
      }
    });
  </script>

  <!-- Code Syntax Highlighting -->
  <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet">
  <script src="/assets/js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>

  <!-- Script Used for Randomizing the Projects Order -->
  <!-- <script type="text/javascript">
    $.fn.shuffleChildren = function() {
      $.each(this.get(), function(index, el) {
        var $el = $(el);
        var $find = $el.children();

        $find.sort(function() {
          return 0.5 - Math.random();
        });

        $el.empty();
        $find.appendTo($el);
      });
    };
    $("#projects").shuffleChildren();
  </script> -->

  <!-- Project Cards Layout -->
  <script type="text/javascript">
    var $grid = $('#projects');

    // $grid.masonry({ percentPosition: true });
    // $grid.masonry('layout');

    // Trigger after images load.
    $grid.imagesLoaded().progress(function() {
      $grid.masonry({ percentPosition: true });
      $grid.masonry('layout');
    });
  </script>

  <!-- Enable Tooltips -->
  <script type="text/javascript">
    $(function () {
      $('[data-toggle="tooltip"]').tooltip()
    })
  </script>

  <!-- Google Analytics -->
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    ga('create', 'UA-197596889-1', 'auto');
    ga('send', 'pageview');
  </script>
    
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZXQXWK2E6C"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-ZXQXWK2E6C');
  </script>
    
  <!-- Baidu Tongji -->  
  <script>
    var _hmt = _hmt || [];
    (function() {
    var hm = document.createElement("script");
    hm.src = "https://hm.baidu.com/hm.js?fc5a69a8465cb0fd34201c2fab055fc4";
    var s = document.getElementsByTagName("script")[0]; 
    s.parentNode.insertBefore(hm, s);
    })();
  </script>

</body>
</html>
