<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <title>Lei Xiong | Translatomer</title>
  <meta name="description" content="">

  <!-- Fonts and Icons -->
  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Helvetica:300,400,500,700|Helvetica:100,300,400,500,700|Material+Icons" />

  <!-- CSS Files -->
  <link rel="stylesheet" href="/assets/css/all.min.css">
  <link rel="stylesheet" href="/assets/css/academicons.min.css">
  <link rel="stylesheet" href="/assets/css/main.css">
  <link rel="canonical" href="/softwares/4_translatomer/">
</head>
<body>
  <!-- Header -->
  <nav id="navbar" class="navbar fixed-top navbar-expand-md grey lighten-5 z-depth-1 navbar-light">
    <div class="container-fluid p-0">
      
        <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Lei</span> Xiong</a>
      
      <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <li class="nav-item ">
            <a class="nav-link" href="/">
              Home
              
            </a>
          </li>
          
            
          
            
          
            
          
            
          
            
          
        </ul>
      </div>
    </div>
  </nav>

  <!-- Scrolling Progress Bar -->
  <progress id="progress" value="0">
    <div class="progress-container">
      <span class="progress-bar"></span>
    </div>
  </progress>

  <!-- Content -->
  <div class="content">
    
  <h1>Translatomer</h1>
  <h6>predicting ribosome profiling reveals translational regulation and interprets disease variants</h6>


<h1 id="translatomer">Translatomer</h1>
<p>This is our implementation for the paper:</p>

<p>Jialin He, Lei Xiong#, Shaohui Shi, Chengyu Li, Kexuan Chen, Qianchen Fang, Jiuhong Nan, Ke Ding, Jingyun Li, Yuanhui Mao, Carles A. Boix, Xinyang Hu, Manolis Kellis, Jingyun Li and Xushen Xiong#. <a href="https://www.nature.com/articles/s42256-024-00915-6">Deep learning prediction of ribosome profiling with Translatomer reveals translational regulation and interprets disease variants</a>.
(Nature Machine Intelligence)</p>

<h2 id="introduction">Introduction</h2>
<p>Translatomer is a transformer-based multi-modal deep learning framework that predicts ribosome profiling track using genomic sequence and cell-type-specific RNA-seq as input.
<img src="https://github.com/xiongxslab/Translatomer/blob/9d26528ab055353b61e7602886099afba1c299ee/img/Model_overview.png" alt="Overview" /></p>

<h2 id="citation">Citation</h2>
<p>If you want to use our codes and datasets in your research, please cite:</p>
<div class="highlighter-rouge"><pre class="highlight"><code>
</code></pre>
</div>

<h2 id="prerequisites">Prerequisites</h2>
<p>To run this project, you need the following prerequisites:</p>
<ul>
  <li>Python 3.9</li>
  <li>PyTorch 1.13.1+cu117</li>
  <li>Other required Python libraries (please refer to requirements.txt)</li>
</ul>

<p>You can install all the required packages using the following command:</p>
<div class="highlighter-rouge"><pre class="highlight"><code>conda create -n pytorch python=3.9.16
conda activate pytorch
</code></pre>
</div>
<div class="language-python highlighter-rouge"><pre class="highlight"><code><span class="n">pip</span> <span class="n">install</span> <span class="n">torch</span><span class="o">==</span><span class="mf">1.13</span><span class="o">.</span><span class="mi">1</span><span class="o">+</span><span class="n">cu117</span> <span class="n">torchvision</span><span class="o">==</span><span class="mf">0.14</span><span class="o">.</span><span class="mi">1</span><span class="o">+</span><span class="n">cu117</span> <span class="n">torchaudio</span><span class="o">==</span><span class="mf">0.13</span><span class="o">.</span><span class="mi">1</span> <span class="o">--</span><span class="n">extra</span><span class="o">-</span><span class="n">index</span><span class="o">-</span><span class="n">url</span> <span class="n">https</span><span class="p">:</span><span class="o">//</span><span class="n">download</span><span class="o">.</span><span class="n">pytorch</span><span class="o">.</span><span class="n">org</span><span class="o">/</span><span class="n">whl</span><span class="o">/</span><span class="n">cu117</span>
</code></pre>
</div>
<div class="language-python highlighter-rouge"><pre class="highlight"><code><span class="n">pip</span> <span class="n">install</span> <span class="o">-</span><span class="n">r</span> <span class="n">requirements</span><span class="o">.</span><span class="n">txt</span> 
</code></pre>
</div>

<h2 id="data-preparation">Data Preparation</h2>
<p>Example data for model training can be downloaded from <a href="https://doi.org/10.5281/zenodo.13751434">Zenodo</a></p>
<ul>
  <li>Put all input files in a <strong>data</strong> folder. The input files have to be organized as follows:
```
    <ul>
      <li>data
        <ul>
          <li>hg38
            <ul>
              <li>K562
                <ul>
                  <li>GSE153597
                    <ul>
                      <li>input_features
++ rnaseq.bw</li>
                      <li>output_features
++ riboseq.bw</li>
                    </ul>
                  </li>
                </ul>
              </li>
              <li>HepG2
                <ul>
                  <li>GSE174419
                    <ul>
                      <li>input_features
++ rnaseq.bw</li>
                      <li>output_features
++ riboseq.bw 
*…
++ gencode.v43.annotation.gff3
++ hg38.fa
++ hg38.fai
++ mean.sorted.bw</li>
                    </ul>
                  </li>
                </ul>
              </li>
            </ul>
          </li>
          <li>mm10
*…
```</li>
        </ul>
      </li>
    </ul>
  </li>
  <li>To generate training data, use the following command:
```
python generate_features_4rv.py [options]</li>
</ul>

<p>[options]:</p>
<ul>
  <li>–assembly  Genome reference for the data. Default = ‘hg38’.</li>
  <li>–celltype  Name of the cell line. Default = ‘K562’.</li>
  <li>–study  GEO accession number for the data. Default = ‘GSE153597’.</li>
  <li>–region_len  The desired sequence length (region length). Default = 65536.</li>
  <li>–nBins  The number of bins for dividing the sequence. Default = 1024.</li>
</ul>

<div class="highlighter-rouge"><pre class="highlight"><code>
Example to run the codes:
</code></pre>
</div>
<p>find data/ -type d -name ‘output_features’ -exec mkdir -p ‘{}/tmp’ \;
find data/ -type d -name ‘input_features’ -exec mkdir -p ‘{}/tmp’ \;
nohup python generate_features_4rv.py –assembly hg38 –celltype HepG2 –study GSE174419 –region_len 65536 –nBins 1024 &amp;
nohup python generate_features_4rv.py –assembly hg38 –celltype K562 –study GSE153597 –region_len 65536 –nBins 1024 &amp;</p>
<div class="highlighter-rouge"><pre class="highlight"><code>
## Model Training
To train the Translatomer model, use the following command:
</code></pre>
</div>
<p>python train_all_11fold.py [options]</p>

<p>[options]:</p>
<ul>
  <li>–seed  Random seed for training. Default value: 2077.</li>
  <li>–save_path  Path to the model checkpoint. Default = ‘checkpoints’.</li>
  <li>–data-root  Root path of training data.  Default = ‘data’ (Required).</li>
  <li>–assembly  Genome assembly for training data. Default = ‘hg38’.</li>
  <li>–model-type  Type of the model to use for training. Default = ‘TransModel’.</li>
  <li>–fold  Which fold of the model training. Default=’0’,</li>
  <li>–patience  Epochs before early stopping. Default = 8.</li>
  <li>–max-epochs  Max epochs for training. Default = 128.</li>
  <li>–save-top-n  Top n models to save during training. Default = 20.</li>
  <li>–num-gpu  Number of GPUs to use for training. Default = 1.</li>
  <li>–batch-size  Batch size for data loading. Default = 32.</li>
  <li>–ddp-disabled  Flag to disable ddp (Distributed Data Parallel) for training. If provided, it will enable DDP with batch size adjustment.</li>
  <li>–num-workers  Number of dataloader workers. Default = 1.
    <div class="highlighter-rouge"><pre class="highlight"><code>Example to run the codes:
</code></pre>
    </div>
    <p>nohup python train_all_11fold.py –save_path results/bigmodel_h512_l12_lr1e-5_wd0.05_ws2k_p32_fold0 –data-root data –assembly hg38 –dataset data_roots_mini.txt –model-type TransModel –fold 0 –patience 6 –max-epochs 128 –save-top-n 128 –num-gpu 1 –batch-size 32 –num-workers 1 &gt;DNA_logs/bigmodel_h512_l12_lr1e-5_wd0.05_ws2k_p32_fold0.log 2&gt;&amp;1 &amp;
nohup python train_all_11fold.py –save_path results/bigmodel_h512_l12_lr1e-5_wd0.05_ws2k_p32_fold1 –data-root data –assembly hg38 –dataset data_roots_mini.txt –model-type TransModel –fold 1 –patience 6 –max-epochs 128 –save-top-n 128 –num-gpu 1 –batch-size 32 –num-workers 1 &gt;DNA_logs/bigmodel_h512_l12_lr1e-5_wd0.05_ws2k_p32_fold1.log 2&gt;&amp;1 &amp;
```</p>
  </li>
</ul>

<h2 id="tutorial">Tutorial</h2>
<ul>
  <li>Load pretrained model
Pretrained model can be downloaded from <a href="https://doi.org/10.5281/zenodo.13751434">Zenodo</a></li>
  <li>An example notebook containing code for applying Translatomer is <a href="https://github.com/xiongxslab/Translatomer/blob/main/Tutorial.ipynb">here</a>.</li>
</ul>

<h2 id="license">License</h2>
<p>This project is licensed under MIT License.</p>

<h2 id="contact">Contact</h2>
<p>For any questions or inquiries, please contact xiongxs@zju.edu.cn.</p>


  </div>

  <!-- Footer -->
  <footer>
    &copy; Copyright 2024 Lei Xiong.
    
    
  </footer>

  <!-- Core JavaScript Files -->
  <script src="/assets/js/jquery.min.js" type="text/javascript"></script>
  <script src="/assets/js/popper.min.js" type="text/javascript"></script>
  <script src="/assets/js/bootstrap.min.js" type="text/javascript"></script>
  <script src="/assets/js/mdb.min.js" type="text/javascript"></script>
  <script async="" src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="sha384-GNFwBvfVxBkLMJpYMOABq3c+d3KnQxudP/mGPkzpZSTYykLBNsZEnG2D9G/X/+7D" crossorigin="anonymous"></script>
  <script src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML"></script>
  <script src="/assets/js/common.js"></script>

  <!-- Scrolling Progress Bar -->
  <script type="text/javascript">
    $(document).ready(function() {
      var navbarHeight = $('#navbar').outerHeight(true);
      $('body').css({ 'padding-top': navbarHeight });
      $('progress-container').css({ 'padding-top': navbarHeight });
      var progressBar = $('#progress');
      progressBar.css({ 'top': navbarHeight });
      var getMax = function() { return $(document).height() - $(window).height(); }
      var getValue = function() { return $(window).scrollTop(); }   
      // Check if the browser supports the progress element.
      if ('max' in document.createElement('progress')) {
        // Set the 'max' attribute for the first time.
        progressBar.attr({ max: getMax() });
        progressBar.attr({ value: getValue() });
    
        $(document).on('scroll', function() {
          // On scroll only the 'value' attribute needs to be calculated.
          progressBar.attr({ value: getValue() });
        });

        $(window).resize(function() {
          var navbarHeight = $('#navbar').outerHeight(true);
          $('body').css({ 'padding-top': navbarHeight });
          $('progress-container').css({ 'padding-top': navbarHeight });
          progressBar.css({ 'top': navbarHeight });
          // On resize, both the 'max' and 'value' attributes need to be calculated.
          progressBar.attr({ max: getMax(), value: getValue() });
        });
      } else {
        var max = getMax(), value, width;
        var getWidth = function() {
          // Calculate the window width as a percentage.
          value = getValue();
          width = (value/max) * 100;
          width = width + '%';
          return width;
        }
        var setWidth = function() { progressBar.css({ width: getWidth() }); };
        setWidth();
        $(document).on('scroll', setWidth);
        $(window).on('resize', function() {
          // Need to reset the 'max' attribute.
          max = getMax();
          setWidth();
        });
      }
    });
  </script>

  <!-- Code Syntax Highlighting -->
  <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet">
  <script src="/assets/js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>

  <!-- Script Used for Randomizing the Projects Order -->
  <!-- <script type="text/javascript">
    $.fn.shuffleChildren = function() {
      $.each(this.get(), function(index, el) {
        var $el = $(el);
        var $find = $el.children();

        $find.sort(function() {
          return 0.5 - Math.random();
        });

        $el.empty();
        $find.appendTo($el);
      });
    };
    $("#projects").shuffleChildren();
  </script> -->

  <!-- Project Cards Layout -->
  <script type="text/javascript">
    var $grid = $('#projects');

    // $grid.masonry({ percentPosition: true });
    // $grid.masonry('layout');

    // Trigger after images load.
    $grid.imagesLoaded().progress(function() {
      $grid.masonry({ percentPosition: true });
      $grid.masonry('layout');
    });
  </script>

  <!-- Enable Tooltips -->
  <script type="text/javascript">
    $(function () {
      $('[data-toggle="tooltip"]').tooltip()
    })
  </script>

  <!-- Google Analytics -->
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    ga('create', 'UA-197596889-1', 'auto');
    ga('send', 'pageview');
  </script>
    
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZXQXWK2E6C"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-ZXQXWK2E6C');
  </script>
    
  <!-- Baidu Tongji -->  
  <script>
    var _hmt = _hmt || [];
    (function() {
    var hm = document.createElement("script");
    hm.src = "https://hm.baidu.com/hm.js?fc5a69a8465cb0fd34201c2fab055fc4";
    var s = document.getElementsByTagName("script")[0]; 
    s.parentNode.insertBefore(hm, s);
    })();
  </script>

</body>
</html>
