<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

  <title>Lei Xiong</title>
  <meta name="description" content="">

  <!-- Fonts and Icons -->
  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Helvetica:300,400,500,700|Helvetica:100,300,400,500,700|Material+Icons" />

  <!-- CSS Files -->
  <link rel="stylesheet" href="http://localhost:4000/assets/css/all.min.css">
  <link rel="stylesheet" href="http://localhost:4000/assets/css/academicons.min.css">
  <link rel="stylesheet" href="http://localhost:4000/assets/css/main.css">
  <link rel="canonical" href="http://localhost:4000/">
</head>
<body>
  <!-- Header -->
  <nav id="navbar" class="navbar fixed-top navbar-expand-md grey lighten-5 z-depth-1 navbar-light">
    <div class="container-fluid p-0">
      
        <div class="row ml-1 ml-sm-0">
          <span class="contact-icon text-center" style="line-height: 1em;">
            <a href="mailto:jsxlei@gmail.com"><i class="fa fa-envelope-square gm-icon"></i></a>
            <a href="https://scholar.google.com/citations?userid=ky6bUTgAAAAJ&hl&user=ky6bUTgAAAAJ&hl" target="_blank" title="Google Scholar"><i class="ai ai-google-scholar-square gs-icon"></i></a>
            <a href="https://www.github.com/jsxlei" target="_blank" title="GitHub"><i class="fab fa-github-square gh-icon"></i></a>
            <a href="https://www.linkedin.com/in/lei-xiong-6776a410b" target="_blank" title="LinkedIn"><i class="fab fa-linkedin li-icon"></i></a>
            <a href="https://www.facebook.com/jsxlei" target="_blank" title="Facebook"><i class="fab fa-facebook-square fb-icon"></i></a>
            <a href="https://twitter.com/jsxlei" target="_blank" title="Twitter"><i class="fab fa-twitter-square tw-icon"></i></a>
          </span>
        </div>
      
      <button class="navbar-toggler ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <li class="nav-item navbar-active font-weight-bold">
            <a class="nav-link" href="http://localhost:4000/">
              Home
              
                <span class="sr-only">(current)</span>
              
            </a>
          </li>
          
            
          
            
          
            
          
            
          
            
          
        </ul>
      </div>
    </div>
  </nav>

  <!-- Scrolling Progress Bar -->
  <progress id="progress" value="0">
    <div class="progress-container">
      <span class="progress-bar"></span>
    </div>
  </progress>

  <!-- Content -->
  <div class="content">
    

<!-- Introduction -->

<div style="display: flex; flex-wrap: wrap;">
    <div class="text-justify p-0">
        <div id="sidebar" class="col-xs-12 col-sm-4 p-0 pt-2 pb-sm-2 pb-4 pl-sm-4 text-center" style="float: right;">
          <img class="profile-img img-responsive" src="http://localhost:4000/assets/img/prof_pic2.jpg" />
          <nav class="sidebar">
            <!-- <ul> -->
              <li class="font-weight-bold"><a href="/assets/pdf/leixiong_cv.pdf">Bio</a></li>
              <li class="font-weight-bold"><a href="#news">News</a></li>
              <li class="font-weight-bold"><a href="#publications">Publications</a></li>
              <li class="font-weight-bold"><a href="#softwares">Softwares</a></li>
              <li class="font-weight-bold"><a href="#education">Education</a></li>
              <li class="font-weight-bold"><a href="#honors and awards">Honors and Awards</a></li>
            <!-- </ul> -->
          </nav>
        </div>
        <h1 class="pb-3 title text-left font-weight-bold">Lei Xiong</h1>
          <p>
            <b>Postdoctorial Associate</b> <br />
            <b>Computer Science and Artificial Intelligence Laboratory (CSAIL) </b> <br /> 
            <b>Massachusetts Institute of Technology (MIT) </b> <br />
            <b>Broad Institute of MIT and Harvard</b> <br />
          </p>
        <p> 
          Currently I am developing algorithms to learn the regulation mechanisms from epigenetics to transcriptomes with paired mulit-omics single-cell datasets since I joined Manolis Kellis Lab.
        </p>
        <p>
          My PhD research focuses on applying artificial intelligence algorithms to develop tools on single-cell genomic data: <a href="https://github.com/jsxlei/SCALE" target="_blank"><b>SCALE</b></a> for clustering single-cell ATAC-seq data to uncover gene regulation in cell heterogeneity, also the first deep learning method and the first imputation method applied on scATAC; the second method is <a href="https://github.com/jsxlei/SCALE" target="_blank"><b>SCALEX</b></a>, for online integration by projecting million-sized single-cell datasets onto the same batch-free biological embedding space via a generalized encoder. 
        </p>
        <p>
          I am also passionate about applying network analysis methods, especially the graph neural network and graph attention network. Previously, I did some research in structural-protein-protein interaction (PPI) network to find out structural modules enriching cancer mutations and drug targets.
        </p>
        <p>
          Before I switched to computational, I had two-year experimental experience in structure biology lab at USTC and Tsinghua when I was an undergraduate student. 
        </p>
        <p>
          <b>I have a broad interest in applying cutting-edge artificial intelligence algorithms including variational auto-encoder, Transformers, diffusion methods to solve biological questions.</b>
        </p>
        <p>
          <b>Feel free to reach out by email jsxlei at gmail.com or other social media listed above!</b>
        </p>
    </div>
</div>

<!-- News -->
<div id="news" class="news mt-3 p-0">
  <h2 class="font-weight-bold">News</h2>
  
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Dec 1, 2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Join <a href="http://compbio.mit.edu/" target="_blank">Prof. <b>Manolis Kellis</b></a> lab in CSAIL at MIT and Broad Institute of MIT and Harvard.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Jun 30, 2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Awarded <b>Outstanding Doctoral Dissertation Award of Tsinghua University</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Jan 13, 2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Awarded <b>Outstanding Graduate of Beijing</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Sep 14, 2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>PhD. thesis Defense.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Feb 15, 2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Our work on  <b>SCALE method for single-cell ATAC-seq analysis via latent feature extraction</b> was selected as <a href="http://gpb.big.ac.cn/news/1010"><b>Top Ten Advances in Bioinformatics in China in 2019</b></a> and 
<a href="http://gpb.big.ac.cn/news/1008"><b>Top Ten Algorithms and Tools for Bioinformatics in China in 2019</b></a> 
by <i>Genomics, Proteomics &amp; Bioinformatics</i>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          Oct 8, 2019
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p>Our work  <b><a href="https://www.nature.com/articles/s41467-019-12630-7#citeas" target="https://www.nature.com/articles/s41467-019-12630-7#citeas">SCALE method for single-cell ATAC-seq analysis via latent feature extraction</a></b> was published on <i>Nature Communications</i>.
</p>
      </div>
    </div>
  
</div>

<!-- publications -->

<div id="publications" class="news mt-3 p-0">
  <h2 class="font-weight-bold">Publications</h2>
  <!-- <nobr><em>*</em></nobr> for equal authorship, <nobr><em>#</em></nobr> for corresponding authorship. -->
  
  <div class="row m-0 p-0" style="border-top: 1px solid #ddd; flex-direction: row-reverse;">
    <div class="col-sm-1 mt-2 p-0 pr-1">
      <h3 class="bibliography-year">2022</h3>
    </div>
    <div class="col-sm-11 p-0">
      <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/scalex.jpg" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <div id="RN528" class="col p-0">
        <h5 class="title mb-0"><a href="https://doi.org/10.1038/s41467-022-33758-z
https://www.nature.com/articles/s41467-022-33758-z.pdf" target="_blank">Online single-cell data integration through projecting heterogeneous datasets into a common cell-embedding space.</a></h5>
      <div class="author">
        
          
            
              
                <nobr><em>Lei Xiong</em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Kang Tian,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yuzhe Li,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Weixi Ning,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Xin Gao,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Qiangfeng Cliff Zhang</nobr>
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>Nature Communications</i></em></nobr>
          
<!--           
              
                <a class="badge font-weight-bold light-green darken-1 align-middle" style="width: 80px;" href="https://www.nature.com/ncomms" target="_blank">
                  Nat Commun
                </a>
              
           -->
          
            2022.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#RN528-abstract" role="button" aria-expanded="false" aria-controls="RN528-abstract">Abstract</a>
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="http://localhost:4000/assets/pdf/scalex/scalex.pdf" target="_blank">PDF</a>
        
        
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="http://localhost:4000https://docs.google.com/presentation/d/18J1EpdZ8NpBP-5l1kiNu_e3o-pdVJ427/edit?usp=sharing&amp;ouid=114907156252810997230&amp;rtpof=true&amp;sd=true" target="_blank">Slides</a>
        
        
          <a class="fab fa-github gh-icon font-weight-light mr-1" href="https://github.com/jsxlei/SCALEX" target="_blank"></a>
        
        
        
        
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="RN528-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            Computational tools for integrative analyses of diverse single-cell experiments are facing formidable new challenges including dramatic increases in data scale, sample heterogeneity, and the need to informatively cross-reference new data with foundational datasets. Here, we present SCALEX, a deep-learning method that integrates single-cell data by projecting cells into a batch-invariant, common cell-embedding space in a truly online manner (i.e., without retraining the model). SCALEX substantially outperforms online iNMF and other state-of-the-art non-online integration methods on benchmark single-cell datasets of diverse modalities, (e.g., single-cell RNA sequencing, scRNA-seq, single-cell assay for transposase-accessible chromatin use sequencing, scATAC-seq), especially for datasets with partial overlaps, accurately aligning similar cell populations while retaining true biological differences. We showcase SCALEX’s advantages by constructing continuously expandable single-cell atlases for human, mouse, and COVID-19 patients, each assembled from diverse data sources and growing with every new data. The online data integration capacity and superior performance makes SCALEX particularly appropriate for large-scale single-cell applications to build upon previous scientific insights.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li>
<li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/jem2022.png" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <div id="10.1084/jem.20211191" class="col p-0">
        <h5 class="title mb-0"><a href="https://doi.org/10.1084/jem.20211191" target="_blank">CD127 imprints functional heterogeneity to diversify monocyte responses in inflammatory diseases.</a></h5>
      <div class="author">
        
          
            
              
                
                  <nobr>Bin Zhang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yuan Zhang,</nobr>
                
              
            
          
        
          
            
              
                <nobr><em>Lei Xiong</em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>Yuzhe Li,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yunliang Zhang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Jiuliang Zhao,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Hui Jiang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Can Li,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yunqi Liu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Xindong Liu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Haofei Liu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yi-Fang Ping,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Qiangfeng Cliff Zhang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Zheng Zhang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Xiu-Wu Bian,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Yan Zhao,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Xiaoyu Hu</nobr>
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>Journal of Experimental Medicine</i></em></nobr>
          
<!--           
              
                <span class="badge font-weight-bold light-green darken-1 align-middle" style="width: 80px;">
                  JEM
                </span>
              
           -->
          
            2022.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#10.1084/jem.20211191-abstract" role="button" aria-expanded="false" aria-controls="10.1084/jem.20211191-abstract">Abstract</a>
        
        
        
        
        
        
        
        
        
        
        
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="10.1084/jem.20211191-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            Inflammatory monocytes are key mediators of acute and chronic inflammation; yet, their functional diversity remains obscure. Single-cell transcriptome analyses of human inflammatory monocytes from COVID-19 and rheumatoid arthritis patients revealed a subset of cells positive for CD127, an IL-7 receptor subunit, and such positivity rendered otherwise inert monocytes responsive to IL-7. Active IL-7 signaling engaged epigenetically coupled, STAT5-coordinated transcriptional programs to restrain inflammatory gene expression, resulting in inverse correlation between CD127 expression and inflammatory phenotypes in a seemingly homogeneous monocyte population. In COVID-19 and rheumatoid arthritis, CD127 marked a subset of monocytes/macrophages that retained hypoinflammatory phenotypes within the highly inflammatory tissue environments. Furthermore, generation of an integrated expression atlas revealed unified features of human inflammatory monocytes across different diseases and different tissues, exemplified by those of the CD127high subset. Overall, we phenotypically and molecularly characterized CD127-imprinted functional heterogeneity of human inflammatory monocytes with direct relevance for inflammatory diseases.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li></ol>   
    </div>
  </div>
  
  <div class="row m-0 p-0" style="border-top: 1px solid #ddd; flex-direction: row-reverse;">
    <div class="col-sm-1 mt-2 p-0 pr-1">
      <h3 class="bibliography-year">2021</h3>
    </div>
    <div class="col-sm-11 p-0">
      <ol class="bibliography"></ol>   
    </div>
  </div>
  
  <div class="row m-0 p-0" style="border-top: 1px solid #ddd; flex-direction: row-reverse;">
    <div class="col-sm-1 mt-2 p-0 pr-1">
      <h3 class="bibliography-year">2019</h3>
    </div>
    <div class="col-sm-11 p-0">
      <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/scale.jpg" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <div id="RN737" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.ncbi.nlm.nih.gov/pubmed/31594952
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6783552/pdf/41467_2019_Article_12630.pdf" target="_blank">SCALE method for single-cell ATAC-seq analysis via latent feature extraction.</a></h5>
      <div class="author">
        
          
            
              
                <nobr><em>L. Xiong</em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>K. Xu,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>K. Tian,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>Y. Shao,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>L. Tang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>G. Gao,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>M. Zhang,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>T. Jiang,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>Q. C. Zhang#</nobr>
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>Nature Communications</i></em></nobr>
          
<!--           
              
                <a class="badge font-weight-bold light-green darken-1 align-middle" style="width: 80px;" href="https://www.nature.com/ncomms" target="_blank">
                  Nat Commun
                </a>
              
           -->
          
            2019.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#RN737-abstract" role="button" aria-expanded="false" aria-controls="RN737-abstract">Abstract</a>
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="http://localhost:4000/assets/pdf/scale/scale.pdf" target="_blank">PDF</a>
        
        
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="http://localhost:4000https://docs.google.com/presentation/d/1wnihWSYj9Oahr97KoJP5t-9sHhhkJwfa01bqYJUgASI/edit?usp=sharing" target="_blank">Slides</a>
        
        
          <a class="fab fa-github gh-icon font-weight-light mr-1" href="https://github.com/jsxlei/SCALE" target="_blank"></a>
        
        
        
        
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="RN737-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            Single-cell ATAC-seq (scATAC-seq) profiles the chromatin accessibility landscape at single cell level, thus revealing cell-to-cell variability in gene regulation. However, the high dimensionality and sparsity of scATAC-seq data often complicate the analysis. Here, we introduce a method for analyzing scATAC-seq data, called Single-Cell ATAC-seq analysis via Latent feature Extraction (SCALE). SCALE combines a deep generative framework and a probabilistic Gaussian Mixture Model to learn latent features that accurately characterize scATAC-seq data. We validate SCALE on datasets generated on different platforms with different protocols, and having different overall data qualities. SCALE substantially outperforms the other tools in all aspects of scATAC-seq data analysis, including visualization, clustering, and denoising and imputation. Importantly, SCALE also generates interpretable features that directly link to cell populations, and can potentially reveal batch effects in scATAC-seq experiments.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li></ol>   
    </div>
  </div>
  
  <div class="row m-0 p-0" style="border-top: 1px solid #ddd; flex-direction: row-reverse;">
    <div class="col-sm-1 mt-2 p-0 pr-1">
      <h3 class="bibliography-year">2015</h3>
    </div>
    <div class="col-sm-11 p-0">
      <ol class="bibliography"><li><div class="row m-0 mt-3 p-0">
  
  <div class="col-sm-4 p-0">
      <div class="ml-0 pl-0">
        <img src="/assets/img/publications/glut3.png" alt="disent" width="240" height="" /> 
      </div>
  </div>
  <div class="col-sm-8 mt-2 mt-sm-0 p-0 pl-xs-0 pl-sm-4 pr-xs-0 pr-sm-2">


    <div id="RN736" class="col p-0">
        <h5 class="title mb-0"><a href="https://www.ncbi.nlm.nih.gov/pubmed/26176916
https://www.nature.com/articles/nature14655.pdf" target="_blank">Molecular basis of ligand recognition and transport by glucose transporters.</a></h5>
      <div class="author">
        
          
            
              
                
                  <nobr>D. Deng,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>P. Sun,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>C. Yan,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>M. Ke,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>X. Jiang,</nobr>
                
              
            
          
        
          
            
              
                <nobr><em>L. Xiong</em>,</nobr>
              
            
          
        
          
            
              
                
                  <nobr>W. Ren,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>K. Hirata,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>M. Yamamoto,</nobr>
                
              
            
          
        
          
            
              
                
                  <nobr>S. Fan,</nobr>
                
              
            
          
        
          
            
              and
              
                
                  <nobr>N. Yan</nobr>
                
              
            
          
        
      </div>

      <div>
        <p class="periodical font-italic">
          
            <nobr><em><i>Nature</i></em></nobr>
          
<!--           
              
                <a class="badge font-weight-bold light-green darken-1 align-middle" style="width: 80px;" href="https://www.nature.com" target="_blank">
                  Nature
                </a>
              
           -->
          
            2015.
          
            
        </p>

      </div>
    
      <div class="col p-0">
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" data-toggle="collapse" href="#RN736-abstract" role="button" aria-expanded="false" aria-controls="RN736-abstract">Abstract</a>
        
        
        
          <a class="badge light-blue waves-effect font-weight-light mr-1" href="http://localhost:4000/assets/pdf/glut3/glut3.pdf" target="_blank">PDF</a>
        
        
        
        
        
        
        
        
        
      </div>
    
      
      <div class="col mt-2 p-0">
        <div id="RN736-abstract" class="collapse">
          <div class="abstract card card-body font-weight-light mr-0 mr-sm-3 p-3">
            The major facilitator superfamily glucose transporters, exemplified by human GLUT1–4, have been central to the study of solute transport. Using lipidic cubic phase crystallization and microfocus X-ray diffraction, we determined the structure of human GLUT3 in complex with D-glucose at 1.5 Å resolution in an outward-occluded conformation. The high-resolution structure allows discrimination of both α- and β-anomers of D-glucose. Two additional structures of GLUT3 bound to the exofacial inhibitor maltose were obtained at 2.6 Å in the outward-open and 2.4 Å in the outward-occluded states. In all three structures, the ligands are predominantly coordinated by polar residues from the carboxy terminal domain. Conformational transition from outward-open to outward-occluded entails a prominent local rearrangement of the extracellular part of transmembrane segment TM7. Comparison of the outward-facing GLUT3 structures with the inward-open GLUT1 provides insights into the alternating access cycle for GLUTs, whereby the C-terminal domain provides the primary substrate-binding site and the amino-terminal domain undergoes rigid-body rotation with respect to the C-terminal domain. Our studies provide an important framework for the mechanistic and kinetic understanding of GLUTs and shed light on structure-guided ligand design.
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
</li></ol>   
    </div>
  </div>
  
</div>

<!--  @*[selected=true]*   -->

<!-- Software -->
<div id="software" class="news mt-3 p-0">
  <h2 class="font-weight-bold">Softwares</h2>
  <div id="softwares" class="row mt-2 pt-3" style="overflow: visible !important;">
    
    
      <div class="project-card">
        
          <a href="http://localhost:4000/softwares/2_scalex/">
        
          <div class="card">
            <img class="card-img-top" src="http://localhost:4000/assets/img/publications/scalex.jpg" alt="project thumbnail" height="150" />
            <div class="card-body">
              <h5 class="card-title">SCALEX</h5>
              <p class="card-text">Online integration of atlas-level single-cell datasets</p>
              <p>
                <!-- <a href="https://github.com/SCALEX" target="_blank"><i class="fab fa-github gh-icon"></i></a> -->
                <a href="https://github.com/jsxlei/SCALEX/stargazers"><img src="https://img.shields.io/github/stars/jsxlei/SCALEX?logo=GitHub&amp;color=yellow" alt="Stars" /></a>
                <a href="https://pypi.org/software/SCALEX"><img src="https://img.shields.io/pypi/v/SCALEX.svg" alt="PyPI" /></a>
                <a href="https://pepy.tech/software/SCALEX"><img src="https://pepy.tech/badge/SCALEX" alt="Downloads" /></a>
              </p>
              <div class="row ml-1 mr-1 p-0">
                
              </div>
            </div>
          </div>
        </a>
      </div>
    
      <div class="project-card">
        
          <a href="http://localhost:4000/softwares/1_scale/">
        
          <div class="card">
            <img class="card-img-top" src="http://localhost:4000/assets/img/publications/scale.jpg" alt="project thumbnail" height="150" />
            <div class="card-body">
              <h5 class="card-title">SCALE</h5>
              <p class="card-text">single-cell ATAC-seq clustering and imputation</p>
              <p>
                <!-- <a href="https://github.com/SCALE" target="_blank"><i class="fab fa-github gh-icon"></i></a> -->
                <a href="https://github.com/jsxlei/SCALE/stargazers"><img src="https://img.shields.io/github/stars/jsxlei/SCALE?logo=GitHub&amp;color=yellow" alt="Stars" /></a>
                <a href="https://pypi.org/software/SCALE"><img src="https://img.shields.io/pypi/v/SCALE.svg" alt="PyPI" /></a>
                <a href="https://pepy.tech/software/SCALE"><img src="https://pepy.tech/badge/SCALE" alt="Downloads" /></a>
              </p>
              <div class="row ml-1 mr-1 p-0">
                
              </div>
            </div>
          </div>
        </a>
      </div>
    
  </div>
<div class="news mt-3 p-0">

<!-- Education-->
<div id="education" class="news mt-3 p-0">
  <h2 class="font-weight-bold">Education</h2>
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2015-2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Tsinghua University</b> <br />
            <b>Ph.D. in Computational Biology</b> <br />
            <b>Advisor</b>: <a href="http://life.tsinghua.edu.cn/lifeen/info/1034/1075.htm" target="_blank"><b>Prof. Qiangfeng Cliff Zhang</b></a> <br />
            <b>Thesis</b>: Artificial intelligence method for single-cell ATAC-seq data via feature extraction
        </p>
      </div>
    </div>
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2011-2015
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>University of Science and Technology of China (USTC)</b> <br />
            <b>B.S. in Biology, Shitsan Pai Talent Program in Life Sciences </b><br />
            <b>Advisor</b>: <a href="https://molbio.princeton.edu/people/nieng-yan" target="_blank"><b>Prof. Nieng Yan</b></a> <br />
            <b>Thesis</b>: Structure basis and transport mechanism of membrane protein GLUT3
        </p>
      </div>
    </div>
</div>


<!-- Honors and Awards-->
<div id="honors and awards" class="news mt-3 p-0">
  <h2 class="font-weight-bold">Honors and Awards</h2>
  
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Outstanding Doctoral Dissertation of Tsinghua University</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2021
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Outstanding Graduate of Beijing</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2020
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>SCALE method for single-cell ATAC-seq analysis via latent feature extraction</b> was selected as <a href="http://gpb.big.ac.cn/news/1010"><b>Top Ten Advances in Bioinformatics in China in 2019</b></a> and 
<a href="http://gpb.big.ac.cn/news/1008"><b>Top Ten Algorithms and Tools for Bioinformatics in China in 2019</b></a> 
by <i>Genomics, Proteomics &amp; Bioinformatics</i>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2019
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Outstanding Scholarship, Advanced Innovation Center of Structure Biology, Tsinghua</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2016
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Innovation Scholarship, Advanced Innovation Center of Structural Biology, Tsinghua</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2013
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>iGEM Gold Medal, USTC-China</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2013
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>2012-2013 Student Scholarship, USTC</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2012
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>2011-2012 Student Scholarship, USTC</b>.
</p>
      </div>
    </div>
  
    <div class="row p-0">
      <div class="col-sm-2 p-0">
        <span class="badge light-blue darken-1 font-weight-bold text-uppercase align-middle date ml-3">
          2011
        </span>
      </div>
      <div class="col-sm-10 mt-2 mt-sm-0 ml-3 ml-md-0 p-0 font-weight-light text">
        <p><b>Freshman Scholarship, USTC</b>.
</p>
      </div>
    </div>
  
</div>
</div></div>


  </div>

  <!-- Footer -->
  <footer>
    &copy; Copyright 2022 Lei Xiong.
    
    
  </footer>

  <!-- Core JavaScript Files -->
  <script src="http://localhost:4000/assets/js/jquery.min.js" type="text/javascript"></script>
  <script src="http://localhost:4000/assets/js/popper.min.js" type="text/javascript"></script>
  <script src="http://localhost:4000/assets/js/bootstrap.min.js" type="text/javascript"></script>
  <script src="http://localhost:4000/assets/js/mdb.min.js" type="text/javascript"></script>
  <script async="" src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="sha384-GNFwBvfVxBkLMJpYMOABq3c+d3KnQxudP/mGPkzpZSTYykLBNsZEnG2D9G/X/+7D" crossorigin="anonymous"></script>
  <script src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML"></script>
  <script src="http://localhost:4000/assets/js/common.js"></script>

  <!-- Scrolling Progress Bar -->
  <script type="text/javascript">
    $(document).ready(function() {
      var navbarHeight = $('#navbar').outerHeight(true);
      $('body').css({ 'padding-top': navbarHeight });
      $('progress-container').css({ 'padding-top': navbarHeight });
      var progressBar = $('#progress');
      progressBar.css({ 'top': navbarHeight });
      var getMax = function() { return $(document).height() - $(window).height(); }
      var getValue = function() { return $(window).scrollTop(); }   
      // Check if the browser supports the progress element.
      if ('max' in document.createElement('progress')) {
        // Set the 'max' attribute for the first time.
        progressBar.attr({ max: getMax() });
        progressBar.attr({ value: getValue() });
    
        $(document).on('scroll', function() {
          // On scroll only the 'value' attribute needs to be calculated.
          progressBar.attr({ value: getValue() });
        });

        $(window).resize(function() {
          var navbarHeight = $('#navbar').outerHeight(true);
          $('body').css({ 'padding-top': navbarHeight });
          $('progress-container').css({ 'padding-top': navbarHeight });
          progressBar.css({ 'top': navbarHeight });
          // On resize, both the 'max' and 'value' attributes need to be calculated.
          progressBar.attr({ max: getMax(), value: getValue() });
        });
      } else {
        var max = getMax(), value, width;
        var getWidth = function() {
          // Calculate the window width as a percentage.
          value = getValue();
          width = (value/max) * 100;
          width = width + '%';
          return width;
        }
        var setWidth = function() { progressBar.css({ width: getWidth() }); };
        setWidth();
        $(document).on('scroll', setWidth);
        $(window).on('resize', function() {
          // Need to reset the 'max' attribute.
          max = getMax();
          setWidth();
        });
      }
    });
  </script>

  <!-- Code Syntax Highlighting -->
  <link href="https://fonts.googleapis.com/css?family=Source+Code+Pro" rel="stylesheet">
  <script src="http://localhost:4000/assets/js/highlight.pack.js"></script>
  <script>hljs.initHighlightingOnLoad();</script>

  <!-- Script Used for Randomizing the Projects Order -->
  <!-- <script type="text/javascript">
    $.fn.shuffleChildren = function() {
      $.each(this.get(), function(index, el) {
        var $el = $(el);
        var $find = $el.children();

        $find.sort(function() {
          return 0.5 - Math.random();
        });

        $el.empty();
        $find.appendTo($el);
      });
    };
    $("#projects").shuffleChildren();
  </script> -->

  <!-- Project Cards Layout -->
  <script type="text/javascript">
    var $grid = $('#projects');

    // $grid.masonry({ percentPosition: true });
    // $grid.masonry('layout');

    // Trigger after images load.
    $grid.imagesLoaded().progress(function() {
      $grid.masonry({ percentPosition: true });
      $grid.masonry('layout');
    });
  </script>

  <!-- Enable Tooltips -->
  <script type="text/javascript">
    $(function () {
      $('[data-toggle="tooltip"]').tooltip()
    })
  </script>

  <!-- Google Analytics -->
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
    ga('create', 'UA-197596889-1', 'auto');
    ga('send', 'pageview');
  </script>
    
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-ZXQXWK2E6C"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-ZXQXWK2E6C');
  </script>
    
  <!-- Baidu Tongji -->  
  <script>
    var _hmt = _hmt || [];
    (function() {
    var hm = document.createElement("script");
    hm.src = "https://hm.baidu.com/hm.js?fc5a69a8465cb0fd34201c2fab055fc4";
    var s = document.getElementsByTagName("script")[0]; 
    s.parentNode.insertBefore(hm, s);
    })();
  </script>

</body>
</html>
